tensorboard/notebook.py

```diff
@@ -278,9 +278,36 @@ def _display(port=None, height=None, print_message=False, display_handle=None):
 
 
 def _display_colab(port, height, display_handle):
-  # TODO(@wchargin): Implement this after merging this code into
-  # google3, where it's easier to develop and test against Colab.
-  raise NotImplementedError()
+  import IPython.display
+  shell = """
+    <script>
+      document.querySelector("base").href = "https://localhost:%s";
+      function fixUpTensorboard() {
+        const tftb = document.querySelector("tf-tensorboard");
+        tftb.removeAttribute("use-hash");
+      }
+      function rehydrate() {
```

Comment:
This seems like kind of an idiosyncratic use of the term?  Could we just call this "executeAllScripts()" or something?  And perhaps with a comment about why this is needed.

Reply:
Sure, `executeAllScripts` is fine with me. You’re right: an explanation
is in order. I’ll add one.